### DISCUSSION TITLE
Linkerd and Nginx Proxy Manager

### DISCUSSION THREAD

**Discussion by zimbres** [linkerd/linkerd2#7796] [Q&A]
I have Nginx Proxy Manager in my cluster, this has no k8s ingress default behavior.

Is it possible the Linkerd inject on it? If I try like a regular deploy, O get 403 on all endpoints.

**Comment by adleong** [comment]
Hi @zimbres.  I'm not able to answer your question without more details and context.  Please feel free to open an issue with as many details as possible, including describing what problem you're trying to solve and the exact behavior that you are seeing.
